### ISSUE TITLE
Task-2 failed

### ISSUE THREAD

**Issue by Mazen001** [Rapptz/discord.py#5206]
`RuntimeError: Task <Task pending name='Task-2' coro=<Client.start() running at C:\Users\Users\AppData\Local\Programs\Python\Python38-32\lib\site-packages\discord\client.py:622> cb=[_run_until_complete_cb() at C:\Users\Users\AppData\Local\Programs\Python\Python38-32\lib\asyncio\base_events.py:184]> got Future <Future pending> attached to a different loop`

When i defined the bot’s variable, task-2 failed from client.py

**Comment by Richie-Moon** [comment]
Whats the code?

**Comment by IHASASUSHI** [comment]
This issue came from me, Here's the code
```py
import logging
import configparser
import asyncio

import discord
from discord.ext.commands import AutoShardedBot

class Zweire(AutoShardedBot):
    def __init__(self, shard_count=None):
        super().__init__(
            command_prefix = ['/'],
            shard_count=shard_count
        )
        self.config = configparser.ConfigParser()
        self.config.read('bot-data.ini')

    def run(self):
        try:
            print("loading cogs")
            cogs = self.config["bot-data"]["cogs"].split(',')
            for cog in cogs:
                super().load_extension("cogs.{cog}".format(cog = cog))
            print("logging in...")
            self.loop.run_until_complete(super().start(self.config["bot-data"]["bot-token"]))
        except KeyboardInterrupt:
            self.loop.run_until_complete(super().logout())
```

**Comment by IHASASUSHI** [comment]
Apparently the issue has fixed itself with 1.4.1 somehow...
